### PR TITLE
Changed links in docs

### PR DESCRIPTION
**Pull request by SZonkil**
closes issue #201

### CODE REVIEW

File: oioioi/amppz/README.rst [hunk 1]
```diff
@@ -1 +1 @@
-Handles `Akademickie Mistrzostwa Polski w Programowaniu Zespołowym type <http://amppz.mimuw.edu.pl/>`_ programming contests.
+Handles `Akademickie Mistrzostwa Polski w Programowaniu Zespołowym type <https://pl.wikipedia.org/wiki/Akademickie_Mistrzostwa_Polski_w_Programowaniu_Zespo%C5%82owym>`_ programming contests.
```
Comment (MasloMaslane) [inline oioioi/amppz/README.rst:1]: The current url for AMPPZ is https://amppz.edu.pl

Comment (DietPawel) [reply oioioi/amppz/README.rst:1]: It's true. In the past it was also http://amppz.tcs.uj.edu.pl/ and http://amppz.ii.uni.wroc.pl/

Comment (MasloMaslane) [reply oioioi/amppz/README.rst:1]: Maybe it’s best to link to Wikipedia since the link changes every few years?

**Comment by DietPawel** [comment]
It does not fully close #201 as there is acm module docs too.

File: oioioi/amppz/README.rst (hunk 1 above)
Comment (DietPawel) [reply oioioi/amppz/README.rst:1]: Let's move our discussion to the issue: https://github.com/sio2project/oioioi/issues/201#issuecomment-1813132670 . Please see my full suggestion there.

**Review by twalen — APPROVED** [review]

**Pull request merged**